doc/mgr/orchestrator.rst

```diff
@@ -102,6 +102,43 @@ Example::
 .. note::
     Output form Ansible orchestrator
 
+
+
+Erase Devices (Zap Devices)
+---------------------------
+
+Erase (zap) a device so it can be re-used.
+
+::
+
+     orch device zap
+
+Example command::
+
+     ceph orch device zap my_hostname my_path true
```

Comment:
```suggestion
     ceph orch device zap my_hostname /dev/sda
```

Reply:
we should be careful with `/dev/sda` in the docs. This is usually the root disk. Even though ceph-volume will probably not be able to zap it, we should avoid copy/paste issues.

Suggesting to use `/dev/sdx` or something